site/_posts/2021-02-16-orbitvu-fusionauth-story.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: blog-post
+title: Orbitvu chose FusionAuth for architectural flexibility, great support, and customizability
+description: Orbitvu, a product photography company, gets more flexibility and value with FusionAuth.
+author: Dan Moore
+image: blogs/become-education-story/become-education-chooses-fusionauth-for-idaas.png
+category: blog
+tags: topic-community-story
+excerpt_separator: "<!--more-->"
+---
+
+Maciej Wisniowski is a FusionAuth community member and team leader in the web development department at Orbitvu, where he is building innovative eCommerce imaging solutions. He chatted with us over email about how he and his team are using FusionAuth to meet their auth needs. 
+
+<!--more-->
+
+*This interview has been lightly edited for clarity and length.*
+
+-------
+
+**Dan:** Tell me a bit about your work as a developer.
+
+**Maciej:** I'm a full-stack web developer working as a Team Leader in Orbitvu, a company that builds innovative imaging solutions for the world of eCommerce. 
+
+Orbitvu provides both software and hardware solutions for automated photography which means our client gets an end-to-end solution that starts with our hardware and desktop software with features like automatic background removal and then is able to effortlessly publish packshots and 360-degree interactive product presentations on the web. The latter part is my responsibility.
```

Comment:
Would it be possible to change "automated photography" to "automated product photography"? Orbitvu is specifically about product photography, would be great if we can also have link there, eg.: [automated product photography](https://orbitvu.com)

Reply:
@pigletto Happy to change the wording to use that text; want to be accurate. 

As for the link, we don't really do tailored linkbacks in blog posts. I did keep a few of your links to other Orbitvu products, hope that is enough :) . If that's an issue, please let me know and I can postpone or cancel the posting of this interview.

Reply:
@mooreds thank you! No problem with the link, forget it :)